### PR TITLE
Regenerate SQR schema with autorest-beta

### PR DESCRIPTION
**Pull request by adibiton**
@anthony-c-martin can you please review?

### CODE REVIEW

File: schemas/2018-04-16/Microsoft.Insights.json [hunk 1]
```diff
@@ -2,21 +2,11 @@
   "id": "https://schema.management.azure.com/schemas/2018-04-16/Microsoft.Insights.json#",
   "$schema": "http://json-schema.org/draft-04/schema#",
   "title": "Microsoft.Insights",
-  "description": "Microsoft Insights Resource Types",
+  "description": "microsoft insights Resource Types",
```
Comment (AntonVasserman) [inline schemas/2018-04-16/Microsoft.Insights.json]: Why change to lowercase?

Comment (adibiton) [reply schemas/2018-04-16/Microsoft.Insights.json]: @anthony-c-martin I can manually change it back to Microsoft Insights - however it is the output from generating with autorest-beta (azureresourceschema version 3.0.67) - wdyt?

Comment (anthony-c-martin) [reply schemas/2018-04-16/Microsoft.Insights.json]: @adibiton This is originating from the provider namespace in the swagger URI paths ('microsoft.insights' is lower-cased) - e.g. https://github.com/Azure/azure-rest-api-specs/blob/30c4f57844896fe41d0f179f5649760dad9101d4/specification/monitor/resource-manager/Microsoft.Insights/stable/2018-04-16/scheduledQueryRule_API.json#L272

I'd recommend you fix this for this file + other similar files in the swagger repo going forwards. For now, manually fixing the casing in this file sounds fine to me.

File: schemas/2018-04-16/Microsoft.Insights.json [hunk 2]
```diff
@@ -7,16 +7,6 @@
     "scheduledQueryRules": {
```
Comment (AntonVasserman) [inline schemas/2018-04-16/Microsoft.Insights.json]: It seems that default values for 'scheduledqueryrules_<alertName>_name' and 'components_<ai/workspaceName>_externalid' aren't generated, in @adibiton example while in the previous version it was generated.
How do we fix it?

File: schemas/2018-04-16/Microsoft.Insights.json [hunk 3]
```diff
@@ -27,38 +17,49 @@
           "type": "string",
           "description": "Resource location"
         },
```
[the remark this answers is not in the document]
Comment (anthony-c-martin) [reply schemas/2018-04-16/Microsoft.Insights.json]: The apiVersion field will always be populated by the ARM service during Export; the missing apiVersion is just a bug in the offline Export simulator tool.

File: schemas/2018-04-16/Microsoft.Insights.json (hunk 2 above)
Comment (anthony-c-martin) [reply schemas/2018-04-16/Microsoft.Insights.json]: Again I think this is just a difference between live & simulated - there's nothing in the schema file which can influence this.

**Pull request merged**